parser.go

```diff
@@ -158,3 +164,23 @@ type CheckStyleError struct {
 	Severity string `xml:"severity,attr,omitempty"`
 	Source   string `xml:"source,attr,omitempty"`
 }
+
+// RDJSONLParser is parser for rdjsonl format.
+type RDJSONLParser struct{}
+
+func NewRDJSONLParser() *RDJSONLParser {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported function NewRDJSONLParser should have comment or be unexported